src/components/views/ViewDataTable/ViewDataTableToolbar.tsx

```diff
@@ -4,23 +4,31 @@ import { Box, Button, Slide, Tooltip } from '@material-ui/core';
 import { FormattedMessage, useIntl } from 'react-intl';
 
 import { ConfirmDialogContext } from 'hooks/ConfirmDialogProvider';
+import ViewDataTableSorting from './ViewDataTableSorting';
+import { GridColDef, GridSortModel } from '@mui/x-data-grid-pro';
 
 export interface ViewDataTableToolbarProps {
     disabled: boolean;
+    gridColumns: GridColDef[];
     isSmartSearch: boolean;
     onColumnCreate: () => void;
     onRowsRemove: () => void;
     onViewCreate: () => void;
     selection: number[];
+    setSortModel: (model: GridSortModel | []) => void;
+    sortModel: GridSortModel | [];
```

Comment:
Because `GridSortModel` is `GridSortItem[]`, you don't need to explicitly state `[]`, since the `GridSortModel` allows an empty list.